core/trino-spi/pom.xml

```diff
@@ -14,6 +14,9 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <!-- The version of SPI to which we try to be compatible with -->
+        <previous.trino.spi.version>375</previous.trino.spi.version>
```

Comment:
This should be auto-derived IMO. Why hard-coded? it adds maintenance overhead.

Note that when the test fails people will be steered first towards maintaining compatibility with an old version, rather than fix the version number here.

Reply:
This is about addressing this comment: https://github.com/trinodb/trino/pull/11722#discussion_r845510571

> Note that when the test fails people will be steered first towards maintaining compatibility with an old version, rather than fix the version number here.

Maybe it is a good thing. Also I can add some comment here to make it more clear what is the goal here. Like `We try to be backward compatible with at least last released version`. WDYT?

Reply:
> Maybe it is a good thing.

This isn't what was intended and is more than we wanted.

>  add some comment here to make it more clear 

the problem is -- the comment here won't be visible; the test failure will

> This is about addressing this comment: https://github.com/trinodb/trino/pull/11722#discussion_r845510571

i replied there